frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -315,11 +333,25 @@ export default {
     },
 
     async getSupplementaryApplications() {
-      this.supplementaryApplications = (await Promise.all(this.applications.map((application) => ApplicationService.getSupplementaryApplications(application.applicationId))))
-        .filter((application) => application.length > 0)
+      this.supplementaryApplications = (await Promise.all(this.applications.map((application) => ApplicationService.getSupplementaryApplications(application?.applicationId))))
+        .filter((application) => application?.length > 0)
         .flat()
     },
 
+    async getRedirectedApplications() {
+      //if an application is redirected - it will not show up in the first call to getActiveApplications() and we will have a mismatch in application length
+      if (this.userInfo?.facilities.length === this.applications.length) return
```

Comment:
I think this check is to make sure don't bother querying Redirected applications if every facility has an application already loaded? What about when we're at the point where there's multiple applications for a facility? I don't think the check is necessary given that the filter below will just return an empty list in this case anyways.

Reply:
true enough.. I'll remove it